app/assets/stylesheets/application.tailwind.css

```diff
@@ -0,0 +1,13 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+/*
```

Comment:
Sprockets::Rails::Helper::AssetNotFound in PublicRecipes#index
Showing /home/enis/recipe-app/app/views/layouts/application.html.erb where line #8 raised:

The asset "tailwind.css" is not present in the asset pipeline.


Optional suggestion make sure to double-check what is happening with tailwind.